src/events/crawler/scrapers/IND/index.js

```diff
@@ -0,0 +1,43 @@
+import * as fetch from '../../lib/fetch.js';
+import * as parse from '../../lib/parse.js';
+import * as rules from '../../lib/rules.js';
+import populationState from './populationState.json';
+
+const scraper = {
+  country: 'IND',
+  url: 'https://www.mohfw.gov.in/',
+  type: 'table',
+  aggregate: 'state',
+
+  async scraper() {
+    this.url = 'https://www.mohfw.gov.in/';
+    const $ = await fetch.page(this.url);
+
+    const $table = $('table[style="font-weight:bold"]');
```

Comment:
It looks like it's in a div with id="cases", shouldn't we select based on that, not font-weight?
```suggestion
    const $table = $('#cases table');
```